docs/general/grants.md

```diff
@@ -7,6 +7,13 @@ keywords: [grants program, grants, funding]
 slug: ../grants
 ---
 
+:::info Decentralized Futures Programme
+
+Please see the [Decentralized Futures Programme](./decentralized-futures.md) before submitting any
```

Comment:
Please change to :

"In addition to the listed grants programs, Web3 Foundation is also currently distributing 20 million USD and 5 million DOT tokens to support individuals and teams as they launch new initiatives aimed at growing the Polkadot ecosystem. For details, see the [Decentralized Futures Programme](./decentralized-futures.md) "

Reply:
added